cluster/gce/config-default.sh

```diff
@@ -176,12 +169,15 @@ ENABLE_METADATA_AGENT="${KUBE_ENABLE_METADATA_AGENT:-none}"
 # Useful for scheduling heapster in large clusters with nodes of small size.
 HEAPSTER_MACHINE_TYPE="${HEAPSTER_MACHINE_TYPE:-}"
 
+LINUX_NODE_LABELS="${KUBE_NODE_LABELS:-}"
```

Comment:
My original PR is to keep an env variable for common node labels, and let `LINUX/WINDOWS_NODE_LABELS` contain only OS-specific labels. There are pros and cons (e.g., code duplication, etc) of course. It's probably more important that we use these OS env variables consistently. Just a note that we should go with one approach and apply it across all env variables.

Reply:
Right, I see what you're saying. Having to split every environment variable into common+Linux+Windows is annoying... do we really need the ability for users to set all three?

I'll leave this as-is for now and revisit it in the full PR with all of the new OS env variables, to ensure consistency as you said.